dubbo-registry/dubbo-registry-xds/src/main/java/org/apache/dubbo/registry/xds/XdsServiceDiscoveryFactory.java

```diff
@@ -23,6 +23,12 @@
 public class XdsServiceDiscoveryFactory extends AbstractServiceDiscoveryFactory {
     @Override
     protected ServiceDiscovery createDiscovery(URL registryURL) {
-        return new XdsServiceDiscovery();
+        XdsServiceDiscovery xdsServiceDiscovery = new XdsServiceDiscovery();
+        try {
+            xdsServiceDiscovery.doInitialize(registryURL);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Do not use printStackTrace to record the exception stack.

Reply:
done